Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -50,7 +50,7 @@ define([
             viewModel._eventHandler.removeInputAction(ScreenSpaceEventType.MOUSE_MOVE);
 
             // Restore hover-over selection to its current value
-            viewModel.picking = viewModel.picking;
+            viewModel.picking = viewModel.picking; // eslint-disable-line no-self-assign
```

Comment:
@ggetz is this code correct?  eslint flagged it because it certainly looks like an error, it was non-obvious to me what is going on here (even with the comment).